Spatial.tscn

```diff
@@ -71,38 +71,34 @@ scale = Vector2( 2, 2 )
 [node name="Line2D" parent="Control/MiniMap" instance=ExtResource( 4 )]
 
 [node name="GlobalMenuButton" type="Control" parent="Control"]
-margin_left = 287.348
-margin_right = 314.348
+margin_left = 321.0
+margin_right = 348.0
 margin_bottom = 32.0
 __meta__ = {
 "_edit_use_anchors_": false
 }
 
 [node name="TextureRect" type="TextureRect" parent="Control/GlobalMenuButton"]
 modulate = Color( 1, 1, 1, 0.439216 )
-margin_left = 1.591
-margin_top = 3.18198
-margin_right = 26.591
-margin_bottom = 28.182
+margin_top = 4.0
+margin_right = 29.0
+margin_bottom = 30.0
 texture = ExtResource( 3 )
 __meta__ = {
 "_edit_use_anchors_": false
 }
 
 [node name="main_menu_toggle" type="Button" parent="Control/GlobalMenuButton"]
 modulate = Color( 1, 1, 1, 0 )
-margin_right = 27.0
-margin_bottom = 32.0
+margin_right = 29.0
+margin_bottom = 30.0
 __meta__ = {
 "_edit_use_anchors_": false
 }
 
 [node name="EditorQuckPanel" type="PanelContainer" parent="Control/GlobalMenuButton"]
 visible = false
 margin_left = 32.0
```

Comment:
I found that the margin besides the left margin does not make a difference if the margin is less than the size. Thus, removing the unnecessary ones